packages/frontend-core/src/utils/login.ts

```diff
@@ -0,0 +1,18 @@
+import { notifications } from "@budibase/bbui"
+
+const SESSIONS_INVALIDATED_KEY = "bb-sessions-invalidated"
+
+export function checkIfSessionsInvalidatedAndNotify() {
+  const invalidatedCount = parseInt(
+    localStorage.getItem(SESSIONS_INVALIDATED_KEY) || "0",
+    10
+  )
+
+  if (invalidatedCount > 0) {
+    const sessionText = invalidatedCount === 1 ? "session" : "sessions"
+    notifications.info(
```

Comment:
I'm not sure this will work in the client library (but maybe it did if it's pushed up?) as it uses its own notification system that doesn't use BBUI notifications. You might need to update this to accept some sort of "notify" param and pass in `notificationStore.actions.info` from the client library instantiation.